### DISCUSSION TITLE
Support kitten remote?

### DISCUSSION THREAD

**Discussion by fecet** [chase/awrit#44] [Q&A] [answered]
It would be great if I can use awrit remotely with kitten ssh, but I failed with a quick attempt

**Comment by chase** [comment, marked as answer]
> It would be great if I can use awrit remotely with kitten ssh, but I failed with a quick attempt

Unfortunately, I really don't have time for the foreseeable future to work on Awrit, but I planned to switch to Electron and Typescript so that it would be easier to contribute.

This would be quite difficult to do well, because there's a lot more to account for regarding lag, missed frames, etc. It's mostly just written assuming that there's local, shared memory to transfer frames. You'd likely be better off using something well established like VNC instead, or forwarding through your SSH connection and connecting your local awrit to that.
